src/types.ts

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export interface Block {
```

Comment:
Are these types duplicate from other repo?

Reply:
There are some shared types with the Wallet Service, yes

But we use all types defined in this file on the sync-daemon, why?

Reply:
I felt I've already seen those. I guess there's no easy way to share these types between the projects, right?

Reply:
Maybe we could create a library to contain types common on Hathor typescript projects, I can create an todo for that on our backlog and an issue here to use it when it is ready. What do you think?

Reply:
Shouldn't we move these types to the `hathor-wallet-lib` for now? I'm assuming that these types are pretty universal and they are useful in several different places.

Reply:
I agree, it would be nice to have all common types centralized somewhere and I think the wallet-lib is a nice place to put them for now